ocaml/utils/warnings.ml

```diff
@@ -1217,6 +1223,11 @@ let message = function
       Printf.sprintf
         "This [@unboxed] attribute cannot be used.\n\
          The type of this value does not allow unboxing."
+  | Mod_by_top modifier ->
+      Printf.sprintf
+        "The modifier %s is a top.\n\
```

Comment:
"is the topmost modifier"? I don't really like "is a top".

Reply:
done